### PR TITLE
Change exporting ARS instead of AGS (DEV-132)

### PR DESCRIPTION
**Pull request by scflode**

### CODE REVIEW

File: export_csv_verwaltungsgebiete.sh [hunk 1]
```diff
@@ -29,46 +28,45 @@ EOF
...
   local query=${2}
-  echo "Starting export for ${ags}"
+  echo "Starting export for ${ars}"
   psql\
     -h localhost\
     -d postgis\
     -U docker\
-    -c "Copy ($query) To '/app/data/${ags}.csv' With CSV DELIMITER ',' HEADER;"
-  echo "Exported to data/${ags}.csv"
+    -c "Copy ($query) To '/app/data/${ars}.csv' With CSV DELIMITER ',' HEADER;"
```
Comment (ctietze) [inline export_csv_verwaltungsgebiete.sh:54]: "/app/data" directory does not exist on first run

- [ ] Add task to create it if it does not exist

**Review by ctietze — CHANGES_REQUESTED** [review]
See file comment

File: import_verwaltungsgebiete.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ if [ ! -f "${ARCHIVE_NAME}.zip" ]; then
 else
   echo "Archive found. Skipping download."
 fi
-unzip "${ARCHIVE_NAME}.zip"
+unzip -o "${ARCHIVE_NAME}.zip"
```
Comment (scflode) [inline import_verwaltungsgebiete.sh:18]: This is added to overwrite already extracted files.

File: export_csv_verwaltungsgebiete.sh [hunk 3]
```diff
@@ -29,46 +28,51 @@ EOF
...
+  echo "Exported to data/${ars}.csv"
+}
+
+function ensure_data_dir {
+  mkdir -p /app/data
```
Comment (scflode) [inline export_csv_verwaltungsgebiete.sh:59]: Normally this folder does exist from the import (if started as mentioned in the readme). But it's more safe like this.

**Review by ctietze — APPROVED** [review]

**Pull request merged**